# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoadRequest.cpp
# Commit: 1751a35aae40
# Full Hash: 1751a35aae40046ff5a7edb4bd43b1fc83b250ff
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move mWebExtGlobal from ScriptLoadRequest to DOMScriptLoadContext; r=jonco,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D136744
# ==============================================================================

diff -r 835c89a0687b -r 1751a35aae40 dom/script/ModuleLoadRequest.cpp
--- a/dom/script/ModuleLoadRequest.cpp	Mon Feb 21 15:39:05 2022 +0000
+++ b/dom/script/ModuleLoadRequest.cpp	Mon Feb 21 15:39:06 2022 +0000
@@ -163,7 +163,8 @@
 
   LOG(("ScriptLoadRequest (%p): Module loaded", this));
 
-  mModuleScript = mLoader->GetFetchedModule(mURI, GetWebExtGlobal());
+  mModuleScript =
+      mLoader->GetFetchedModule(mURI, GetLoadContext()->GetWebExtGlobal());
   if (!mModuleScript || mModuleScript->HasParseError()) {
     ModuleErrored();
     return;